### PR TITLE
fswebcam, wacaw: add pages

### PR DESCRIPTION
**Pull request by reactorcoremeltdown**

### CODE REVIEW

**Comment by J053Fabi0** [comment]
## This command is no longer available on mac os x

File: pages/common/fswebcam.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#FSWEBCAM
```
Comment (sbrl) [inline pages/common/fswebcam.md]: Could this be in lowercase, please?

File: pages/common/sshfs.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# SSHFS
```
Comment (sbrl) [inline pages/common/sshfs.md]: Same here - could this be in lowercase, please?

File: pages/common/fswebcam.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+> small and simple webcam for *nix
```
Comment (sbrl) [inline pages/common/fswebcam.md]: Perhaps `A small and simple` would read better?

File: pages/common/sshfs.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+> filesystem client based on ssh
```
Comment (sbrl) [inline pages/common/sshfs.md]: Perhaps `A filesystem` would read better?

File: pages/common/sshfs.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+- mounting remote directory
```
Comment (sbrl) [inline pages/common/sshfs.md]: Descriptions of a command ideally should start with a capital letter, and end in a colon. Perhaps travis-ci crashed - it should have warned about this :P

File: pages/common/tee.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#TEE
```
Comment (sbrl) [inline pages/common/tee.md]: We already have a page about `tee` under osx and linux, as windows does not currently support `tee` by default. Perhaps you could merge your examples with those pages?

File: pages/osx/wacaw.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+#WACAW
```
Comment (sbrl) [inline pages/osx/wacaw.md]: Same here. Could this be in lowercase please?

File: pages/osx/wacaw.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+> a little command-line tool for Mac OS X that allows you to capture both still pictures and video from an attached camera.
```
Comment (sbrl) [inline pages/osx/wacaw.md]: Perhaps a capital letter at the beginning would read better?

File: pages/osx/wacaw.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+- Take a picture from webcam
```
Comment (sbrl) [inline pages/osx/wacaw.md]: Same here. Could all command descriptions end in a colon please?

**Pull request merged**